builtin/rebase--helper.c

```diff
@@ -98,5 +101,7 @@ int cmd_rebase__helper(int argc, const char **argv, const char *prefix)
 		return !!edit_todo_list(flags);
 	if (command == SETUP_REFLOG && argc > 1)
 		return !!setup_reflog_action(&opts, argv[1], verbose);
+	if (command == CHECKOUT_ONTO && argc > 3)
```

Comment:
What happens if argc is 5 or 6 for example?

Reply:
It still runs. Is this a real problem? We just ignore them.

Reply:
Yeah, but I think we prefer to die in this case as it means that there is probably an issue in the calling shell script.

Reply:
So the `argc > 3` should probably be `argc == 4`.

Reply:
(sorry, when argc is 5, it’s when rebase--helper was called with the `--verbose` flag)

Reply:
Right.

Reply:
I think that the function that parses options removes the flags it interprets.